src/Symfony/Component/Yaml/Parser.php

```diff
@@ -804,4 +812,44 @@ private function isBlockScalarHeader()
     {
         return (bool) preg_match('~'.self::BLOCK_SCALAR_HEADER_PATTERN.'$~', $this->currentLine);
     }
+
+    /**
+     * Returns true if the current line is a collection item.
+     *
+     * @return bool Returns true if the current line is a collection item line, false otherwise.
```

Comment:
The description of the returnt ype here is not needed (it's already clear from the method's description), just use `@return true`.